### PR TITLE
Tweak VSCode config

### PR DESCRIPTION
**Pull request by AlekSi**
Extension ID changed.

### CODE REVIEW

**Comment by AlekSi** [comment]
@askomorokhov @palash25 please check if it works for you. That's not urgent, and doesn't have to be done in this sprint :)

**Comment by BupycHuk** [comment]
I don't use vscode, so I can't review this PR :)

**Comment by AlekSi** [comment]
But as a code owner you have too 😝

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -21,20 +21,49 @@
...
+		"go.formatTool": "goimports",
+		"go.liveErrors": {
+			"enabled": true,
+			"delay": 500
 		},
+		// "go.lintOnSave": "workspace",
+		"go.lintOnSave": "package",
+		"go.lintTool": "golangci-lint",
+		"go.lintFlags": [
+			"--fast"
```
Comment (askomorokhov) [inline .devcontainer/devcontainer.json:64]: Only once I received timeout error - cannot catch it again.
(longer `--timeout` as the future fix can help)

**Pull request merged**